catroweb.yaml

```diff
@@ -1162,4 +1257,99 @@ components:
         filesize:
           type: number
           example: 0.14381762458251943
-          description: filesize in megabytes
\ No newline at end of file
+          description: filesize in megabytes
+
+    NotificationsId:
+      type: integer
+      minimum: 0
+      description: Id of a notification
+      default: 0
+      example: 2
+
+    NotificationsType:
+      type: string
+      enum:
+        - all
+        - reaction
+        - follow
+        - comment
+        - remix
+      default: all
+
+    NotificationsCountResponse:
+      type: object
+      properties:
+        total:
+          type: integer
+          example: 9
+        like:
+          type: integer
+          example: 1
+        follower:
+          type: integer
+          example: 0
+        comment:
+          type: integer
+          example: 5
+        remix:
+          type: integer
+          example: 3
+
+    NotificationsResponse:
+      type: array
+      items:
+        $ref: '#/components/schemas/NotificationResponse'
+
+    NotificationResponse:
+      type: object
+      properties:
+        id:
+          type: integer
+          example: 123
+          description: Id of the notification
+        type:
+          type: string
+          example: remix
+          description: Notification Type
+        seen:
+          type: boolean
+          description: Seen status of the notification
+        content:
+          type: array
+          items:
+            $ref: '#/components/schemas/NotificationContent'
+
+    NotificationContent:
+      type: object
+      properties:
+        from:
+          $ref: '#/components/schemas/Uuid'
+        from_name:
+          type: string
+          example: Catroweb
+          description: Username of the user who caused the notification
+        program:
```

Comment:
Even with UUIDs a unique description would be nice to describe the field to other teams.
![image](https://user-images.githubusercontent.com/40868718/90860533-779d2200-e38a-11ea-9e6d-f5c9eaba4e5e.png)

Reply:
A possible workaround is to wrap $ref into allOf - this can be used to "add" attributes to a $ref but not override existing attributes.